designer/client/src/components/graph/node-modal/editors/expression/CustomAceEditorCompleter.tsx

```diff
@@ -30,11 +30,21 @@ function isSpelTokenAllowed(iterator, modeId): boolean {
     return modeId === "ace/mode/spel" || modeId === "ace/mode/spelTemplate";
 }
 
-interface Editor extends Ace.Editor {
-    readonly completer: {
-        activated: boolean;
-        openPopup: (editor: Editor, prefix: string, keepPopupPosition: boolean) => void;
-    };
+declare module "ace-builds" {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Ace {
+        interface Autocomplete {
+            base: Ace.Point;
+            editor: Ace.Editor;
+            activated: boolean;
```

Comment:
Do we need it? looks like it's not called and if i understand correctly, we extend predefined Ace.Autocomplete interface here

Reply:
this is not an extension but more supplement to what was delivered with holes. maybe it's not used by us anymore, maybe I used it while debugging, but it's identified already and could help in future

Reply:
Ok, I thought there was our default propose which we use to extend the Autocomplete interface, but it it's existing Autocomplete props, that's ok.